src/test/java/uk/gov/hmcts/reform/civil/helpers/sdo/SdoHelperTest.java

```diff
@@ -216,7 +216,7 @@ void shouldReturnLocationForNIHL(boolean isAltLoc) {
         }
 
         @ParameterizedTest
-        @ValueSource(booleans = {true, false})
+        @ValueSource(booleans = {true})
```

Comment:
ParameterizedTest passing only one value?

Reply:
Makes sense. Removed the ParameterizedTest and added individual tests.